app/views/pictures/index.html.haml

```diff
@@ -0,0 +1,29 @@
+- if @pictures.any?
+  - @pictures.reverse.each do |picture|
+    %div.img_container
+      %ul.img_info
+        %li.username= picture.user.username
+        %li.timestamp= picture.timestamp
+      = image_tag picture.image.url(:medium)
+      %div.caption_container
```

Comment:
`%div.caption_container` can be written as `.caption_container` since `%div` is implicit